Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/README.md

```diff
@@ -0,0 +1,154 @@
+# Malware Intelligence connector
+
+## Table of contents
+
+1. [Overview](#overview)
+2. [Prerequisites](#prerequisites)
+3. [Deployment instructions](#deployment-instructions)
+4. [Post-deployment instructions](#post-deployment-instructions)
+5. [Querying Intel 471 Malware Intelligence data in Sentinel](#querying-intel-471-malware-intelligence-data-in-sentinel)
+6. [Data mapping](#data-mapping)
+7. [Script for granting ThreatIndicators.ReadWrite.OwnedBy role](#script-for-granting-threatindicatorsreadwriteownedby-role)
+
+## Overview
+
+This connector fetches malware intelligence indicators from the Intel 471's Titan API and ingests them directly
```

Comment:
the way the template is looking it's a playbook, so kindly mention. "playbook" instead of connector .